### PR TITLE
Fix invalid session regeneration on wrong session ID

### PR DESCRIPTION
**Pull request by fredericgboutin-yapla**
Fix for https://github.com/zf1s/zf1/issues/165

### CODE REVIEW

File: packages/zend-session/library/Zend/Session.php [hunk 1]
```diff
@@ -426,7 +426,7 @@ public static function start($options = false)
             // Generate a valid, temporary replacement
             self::setId(md5(self::getId()));
             // Force a regenerate after session is started
-            self::$_regenerateIdState = -1;
+			self::regenerateId();
```
[the remark this answers is not in the document]
Comment (glensc) [reply packages/zend-session/library/Zend/Session.php:429]: the ident seems off: mixing tabs and spaces?

stick to existing formatting.

also, never send pull request from `master` branch. all commits added to the branch end up in the pull request which might not be wanted. also rebase force push might be difficult (rejected)

[the remark this answers is not in the document]
Comment (fredericgboutin-yapla) [reply packages/zend-session/library/Zend/Session.php]: 🤔 I have editorconfig up and running, though 🤷 

I meant to checkout a branch, my bad (again).

[the remark this answers is not in the document]
Comment (glensc) [reply packages/zend-session/library/Zend/Session.php]: you didn't need to close it yet, if it's not conflicting yet. i.e you do not yet want other changes to your master.

and if you did create a replacement pr, mention it in old pr too to keep things transparent

[the remark this answers is not in the document]
Comment (fredericgboutin-yapla) [reply packages/zend-session/library/Zend/Session.php]: It closed automatically as soon as I rolled back my `master` branch to follow yours.

**Pull request closed without merging**